spec/requests/admin/creator_settings_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+RSpec.describe "/creator_settings/new", type: :request do
+  let!(:super_admin) { create(:user, :super_admin) }
+  let!(:non_admin_user) { create(:user) }
+  let(:params) do
+    { community_name: "Climbing Life",
+      logo_svg: "https://dummyimage.com/300x300.png",
+      primary_brand_color_hex: "000000",
+      public: true,
+      invite_only: false }
+  end
+
+  before do
+    allow(FeatureFlag).to receive(:enabled?).with(:creator_onboarding).and_return(true)
+    allow(Settings::General).to receive(:waiting_on_first_user).and_return(false)
+  end
+
+  describe "GET /admin/creator_settings/new" do
+    before do
+      sign_in super_admin
+      get new_admin_creator_setting_path
+    end
+
+    context "when the user is a super admin" do
+      it "allows the request" do
+        expect(response).to have_http_status(:ok)
+      end
+
+      xit "renders the correct page" do
+        expect(response.body).to include("Lovely! Let's set up your Forem.")
+      end
```

Comment:
This is currently failing--it doesn't appear that the view is rendering as expected. 🤔